app/components/Dashboard/programAdministration.tsx

```diff
@@ -31,21 +31,26 @@ const ProgramDataManagement: React.FunctionComponent<Props> = ({
   return (
     <Card className="admin-control-card">
       <Card.Body>
-        <Card.Title>Program and Data management</Card.Title>
+        <Card.Title>Program Administration</Card.Title>
         <Card.Text>
-          Update program parameters, view program insights etc.
+          Review applications and administer program details:
         </Card.Text>
       </Card.Body>
       <ListGroup variant="flush">
-        {EditableItems}
+        <ListGroup.Item>
+          <Link href="/analyst/applications">
+            <Card.Link href="#">Submitted applications</Card.Link>
+          </Link>
+        </ListGroup.Item>
         <ListGroup.Item>
           <Card.Link
             target="_blank"
             href="https://cas-metabase.pathfinder.gov.bc.ca/"
```

Comment:
This link will need updating, and probably should be in an environment variable, if we want it to point to the dev/test/prod metabase instance.